docs/concepts/configuration/assign-pod-node.md

```diff
@@ -190,13 +190,7 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
-In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limited set of topology keys are allowed.
-By default the allowed topology keys are:
-
-* `kubernetes.io/hostname`
-* `failure-domain.beta.kubernetes.io/zone`
-* `failure-domain.beta.kubernetes.io/region`
+The `topologyKey` can be any legal label value.
```

Comment:
At first, I'd thought to replace the world "allowed" with "recommended", It looks better, but I  don't know anything about the implementation, so I have no idea whether it would get better performance or not. Then I had to remove it completely, to be prudent